### ISSUE TITLE
get_ncaa_game_logs

### ISSUE THREAD

**Issue by robert-frey** [BillPetti/baseballr#190]
Getting this error when running the function: 
Error: Can't subset columns that don't exist.
x Column `G` doesn't exist.
Looked at the code and the issue is this:

Line 54 and 83,
names(payload_df) <- payload_df[3,]

The names are actually in the second row, resulting in:
names(payload_df) <- payload_df[2,]

**Comment by daviddalpiaz** [comment]
Can you provide a reproducible example that generates this error? Running the example code from the documentation does not trigger this error.

```r
library(baseballr)
get_ncaa_game_logs(player_id = 1879650, year = 2019, type = "batting")
```

**Comment by BillPetti** [comment]
This should be fixed via [776a76dc8f81ecbd3bf857f20f108f671026a11d](https://github.com/BillPetti/baseballr/commit/776a76dc8f81ecbd3bf857f20f108f671026a11d)
